### PR TITLE
Restore the format of quantum/rgblight_breathe_table.h

### PR DESCRIPTION
**Pull request by mtei**
## Description

Restore the format of quantum/rgblight_breathe_table.h before applying clang-format. Because this file is machine generated, I want to keep the generated result as it is.

**This commit does not change the build result.**

## Types of Changes

- [x] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: quantum/rgblight_breathe_table.h [hunk 1]
```diff
@@ -1,466 +1,115 @@
 #ifndef RGBLIGHT_EFFECT_BREATHE_TABLE
```
Comment (fauxpark) [inline quantum/rgblight_breathe_table.h]:
```suggestion
#pragma once
```

File: quantum/rgblight_breathe_table.h [hunk 2]
```diff
@@ -1,466 +1,115 @@
...
 #define RGBLIGHT_EFFECT_BREATHE_TABLE
```
Comment (fauxpark) [inline quantum/rgblight_breathe_table.h]:
```suggestion

```

File: quantum/rgblight_breathe_table.h [hunk 3]
```diff
@@ -1,466 +1,115 @@
...
+// clang-format off
 
 const uint8_t rgblight_effect_breathe_table[] PROGMEM = {
-/* #define RGBLIGHT_EFFECT_BREATHE_CENTER   1.85 */
-/* #define RGBLIGHT_EFFECT_BREATHE_MAX      255 */
+  /* #define RGBLIGHT_EFFECT_BREATHE_CENTER   1.85 */
```
Comment (fauxpark) [inline quantum/rgblight_breathe_table.h]:
```suggestion
    /* #define RGBLIGHT_EFFECT_BREATHE_CENTER   1.85 */
```

File: quantum/rgblight_breathe_table.h [hunk 4]
```diff
@@ -1,466 +1,115 @@
...
+  /* #define RGBLIGHT_EFFECT_BREATHE_MAX      255 */
```
Comment (fauxpark) [inline quantum/rgblight_breathe_table.h]:
```suggestion
    /* #define RGBLIGHT_EFFECT_BREATHE_MAX      255 */
```

File: util/rgblight_breathing_table_calc.c [hunk 5]
```diff
@@ -43,7 +44,7 @@ int main(void) {
         printf(" #endif /* %d bytes table */\n", s == 0 ? 256:(s== 1 ? 128: 64));
     }
     printf("};\n");
-    printf("\nstatic const int table_scale = 256/sizeof(rgblight_effect_breathe_table);\n");
+    printf("\nstatic const int table_scale = 256 / sizeof(rgblight_effect_breathe_table);\n");
     printf("\n#endif /* RGBLIGHT_EFFECT_BREATHE_TABLE */\n");
```
Comment (fauxpark) [inline util/rgblight_breathing_table_calc.c]:
```suggestion
```

File: util/rgblight_breathing_table_calc.c [hunk 6]
```diff
@@ -27,7 +27,8 @@ int main(void) {
               );
     }
     printf("#ifndef RGBLIGHT_EFFECT_BREATHE_TABLE\n");
```
Comment (fauxpark) [inline util/rgblight_breathing_table_calc.c]:
```suggestion
    printf("#pragma once\n\n");
```

File: util/rgblight_breathing_table_calc.c [hunk 7]
```diff
@@ -27,7 +27,8 @@ int main(void) {
...
-    printf("#define RGBLIGHT_EFFECT_BREATHE_TABLE\n\n");
+    printf("#define RGBLIGHT_EFFECT_BREATHE_TABLE\n");
```
Comment (fauxpark) [inline util/rgblight_breathing_table_calc.c]:
```suggestion
```

File: util/rgblight_breathing_table_calc.c [hunk 8]
```diff
@@ -27,7 +27,8 @@ int main(void) {
...
+    printf("// clang-format off\n\n");
     printf("const uint8_t rgblight_effect_breathe_table[] PROGMEM = {\n");
     printf("  /* #define RGBLIGHT_EFFECT_BREATHE_CENTER   %.2f */\n", RGBLIGHT_EFFECT_BREATHE_CENTER);
```
Comment (fauxpark) [inline util/rgblight_breathing_table_calc.c]:
```suggestion
    printf("    /* #define RGBLIGHT_EFFECT_BREATHE_CENTER   %.2f */\n", RGBLIGHT_EFFECT_BREATHE_CENTER);
```

File: util/rgblight_breathing_table_calc.c [hunk 9]
```diff
@@ -27,7 +27,8 @@ int main(void) {
...
     printf("  /* #define RGBLIGHT_EFFECT_BREATHE_MAX      %d */\n", RGBLIGHT_EFFECT_BREATHE_MAX);
```
Comment (fauxpark) [inline util/rgblight_breathing_table_calc.c]:
```suggestion
    printf("    /* #define RGBLIGHT_EFFECT_BREATHE_MAX      %d */\n", RGBLIGHT_EFFECT_BREATHE_MAX);
```

File: quantum/rgblight_breathe_table.h (hunk 1 above)
Comment (drashna) [reply quantum/rgblight_breathe_table.h]: Actually, I think this is intentional: 
https://github.com/qmk/qmk_firmware/blob/e318d2ca1554a80e1a407bf263d5c55da33afbe2/quantum/rgblight.c#L857-L858

Comment (fauxpark) [reply quantum/rgblight_breathe_table.h]: Fair enough; deleted the suggestions removing this define (but kept the pragma once).

**Comment by tzarc** [comment]
What's left outstanding here?

**Comment by fauxpark** [comment]
I think I'd prefer to see this as a Python script; then no compilation is required.

**Comment by tzarc** [comment]
Retargeted to `develop` due to core changes, marking as "on hold" until we figure out whether we're going down the python route.

**Pull request closed without merging**